app/views/tasks/index.html.haml

```diff
@@ -19,5 +22,12 @@
         %td=link_to 'Destroy', task, :method => :delete, :data => { :confirm => 'Are you sure?' }
 
 %br
-
+%br
 =link_to 'Add a task', new_task_path
+%br
+- if params["completed"]=="true" && Task.where(completed: false).size()>0
```

Comment:
What is your thought process behind hiding the incomplete/completed tasks links if there are none?

Reply:
My thought process was to hide if there was no incomplete/completed tasks since a user doesn't need to view an empty page. However, I can of course modify it

Reply:
That makes sense to me, no need for an unnecessary click. Do you have any thoughts on adding queries like this to the view? Are there any other places you considered putting it?